### PR TITLE
Add RUN_ON_STARTUP for convenience and testing

### PR DESCRIPTION
**Pull request by ThomDietrich**
Hey @djmaze,
I was missing this simple feature during set-up and testing.

Best!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,9 +64,10 @@ E.g.
 
 * `BACKUP_CRON`- A cron expression for when to run the backup. E.g. `0 30 3 * * *` in order to run every night at 3:30 am. See [the go-cron documentation](https://godoc.org/github.com/robfig/cron) for details on the expression format
 * `PRUNE_CRON` - A cron expression for when to run the prune job. E.g. `0 0 4 * * *` in order to run every night at 4:00 am. See [the go-cron documentation](https://godoc.org/github.com/robfig/cron) for details on the expression format
-* `RESTIC_REPOSITORY` - location of the restic repository. You can use [any target supported by restic](https://restic.readthedocs.io/en/stable/030_preparing_a_new_repo.html). Default `/mnt/restic`
-* `RESTIC_BACKUP_SOURCES` - source directory to backup. Make sure to mount this into the container as a volume (see the example configs). Default `/data`
-* `RESTIC_PASSWORD` - password for the restic repository. Will also be used to initialize the repository if it is not yet initialized
+* `RUN_ON_BOOT` - Optional. Execute the backup or prune job right on startup, in addition to the given con expression. Set to any value to take effect
```
Comment (djmaze) [inline README.md:67]: `cron`

**Comment by djmaze** [comment]
I just had a second thought about this. As the word *boot* has no meaning in the concept of a container, I would prefer to name the variable `RUN_ON_STARTUP`. (That's also how you explained it, so.. ;)) What do you think?

**Comment by ThomDietrich** [comment]
Hey! The inspiration for the feature and the variable name came from another popular restic docker image. I totally agree that "boot" is not the right term. PR changed ;)

**Comment by djmaze** [comment]
@ThomDietrich Sounds good to me, go ahead.

**Comment by ThomDietrich** [comment]
Ready to merge

**Comment by djmaze** [comment]
@ThomDietrich The commit history looks really ugly now. Could you squash commits please?

**Comment by djmaze** [comment]
Also please, instead of *Update file.xyz* please make [a descriptive comment](https://juffalow.com/other/write-good-git-commit-message) on what your change is about.

**Comment by ThomDietrich** [comment]
I totally agree. That's also why I didn't care about the commit messages. You can select "squash and merge" in the PR merge options. That's what I would normally do in my projects 😉  Let me know if I misunderstood you 👍 

image

**Comment by zoispag** [comment]
Probably rebase is what @djmaze meant.
Sure, the PR can be squashed and merged, but it’s good to have a descriptive history of commits when looking an open source PR on GitHub. :)

**Comment by djmaze** [comment]
Well, in the case of this PR, regarding the content we now have two different changes which each deserve a commit: 

* the addition on the RUN_ON_STARTUP flag (including the documentation)
* the additional check for the two conflicting cron variables

I can go ahead and change it accordingly.

**Comment by ThomDietrich** [comment]
Fair point. Let me sort this out.
The simple reason for the messy commit history is the fact, that I am not on my device atm and am using git/GitHub through the website 😅

**Comment by djmaze** [comment]
Thanks :+1:

**Pull request merged**